ivy-data/src/main/java/com/ivy/data/db/migration/Migration124to125_LoanEditDateTime.kt

```diff
@@ -0,0 +1,20 @@
+package com.ivy.data.db.migration
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+class Migration124to125_LoanEditDateTime : Migration(124, 125) {
+    companion object {
+        private const val LOANS_TABLE = "loans"
+        private const val LOANS_NEW_TABLE = "loans_new"
+    }
+
+    override fun migrate(db: SupportSQLiteDatabase) {
```

Comment:
Ahh, this migration seems complicated and risky. Is there a better way?

Reply:
Why though? SQLite doesn't support editing column datatypes, unlike SQL. And the table name remains "loans" as previous, as it gets renamed just after copying the existing data from the old table.